integration-tests/src/test/resources/wdt-models/bad-model-file.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2020, 2021, Oracle and/or its affiliates.
```

Comment:
copyright (c) 2020, 2021, Oracle and/or its affiliates. --> copyright (c)  2021, Oracle and/or its affiliates.

Reply:
changed